### PR TITLE
core: remove policy default values initialisation

### LINKED ISSUE #1534: Make policy contract initialization compatible with preview4 state of C# node
As neo-project/neo#1867 certainly won't be a part of preview4 we need to do the opposite of what's described there (neo-project/neo#1795 essentially) to be compatible with C# node.

### PR DESCRIPTION
**Pull request by AnnaShaleva**
Close #1534.

### CODE REVIEW

File: pkg/core/native/policy.go [hunk 1]
```diff
@@ -493,16 +499,21 @@ func (p *Policy) setUint32WithKey(dao dao.DAO, key []byte, value uint32) error {
...
 	if si == nil {
-		return 0
+		return 0, errors.New("key not found")
 	}
-	return int64(binary.LittleEndian.Uint64(si.Value))
+	return int64(binary.LittleEndian.Uint64(si.Value)), nil
 }
 
 func (p *Policy) setInt64WithKey(dao dao.DAO, key []byte, value int64) error {
 	si := dao.GetStorageItem(p.ContractID, key)
```
Comment (roman-khimov) [inline pkg/core/native/policy.go:511]: Why do we `Get` when we're setting?

File: pkg/core/native/policy.go [hunk 2]
```diff
@@ -475,16 +476,21 @@ func (p *Policy) unblockAccount(ic *interop.Context, args []stackitem.Item) stac
...
 	if si == nil {
-		return 0
+		return 0, errors.New("key not found")
 	}
-	return binary.LittleEndian.Uint32(si.Value)
+	return binary.LittleEndian.Uint32(si.Value), nil
 }
 
 func (p *Policy) setUint32WithKey(dao dao.DAO, key []byte, value uint32) error {
 	si := dao.GetStorageItem(p.ContractID, key)
```
Comment (roman-khimov) [inline pkg/core/native/policy.go:488]: Same thing here.

File: pkg/core/native/policy.go [hunk 3]
```diff
@@ -200,16 +173,28 @@ func (p *Policy) OnPersistEnd(dao dao.DAO) error {
 	p.lock.Lock()
 	defer p.lock.Unlock()
 
-	maxTxPerBlock := p.getUint32WithKey(dao, maxTransactionsPerBlockKey)
+	maxTxPerBlock, err := p.getUint32WithKey(dao, maxTransactionsPerBlockKey)
```
Comment (roman-khimov) [inline pkg/core/native/policy.go:176]: Maybe we don't even need these `maxTxPerBlock` and similar variables?

File: pkg/core/native/policy.go [hunk 4]
```diff
@@ -475,16 +476,21 @@ func (p *Policy) unblockAccount(ic *interop.Context, args []stackitem.Item) stac
 	return stackitem.NewBool(true)
 }
 
-func (p *Policy) getUint32WithKey(dao dao.DAO, key []byte) uint32 {
+func (p *Policy) getUint32WithKey(dao dao.DAO, key []byte) (uint32, error) {
```
Comment (roman-khimov) [inline pkg/core/native/policy.go:479]: What if we're to have
```
func (p *Policy) getUint32WithKey(dao dao.DAO, key []byte, default uint32) uint32
```
I think it'd simplify a lot of code using this function.

**Review by roman-khimov — CHANGES_REQUESTED** [review]

File: pkg/core/native/policy.go (hunk 4 above)
Comment (AnnaShaleva) [reply pkg/core/native/policy.go]: Fixed, looks prettier now.

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**